packages/components/tests/dummy/app/styles/pages/db-gutters.scss

```diff
@@ -0,0 +1,47 @@
+// GUTTERS
+
+// Loose spacing scale, move out to centralize for re-use:
+// (See: https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?node-id=2008%3A3)
+:root {
+    --hds-spacing-xxs: 4px;
+    --hds-spacing-xs: 8px;
+    --hds-spacing-sm: 12px;
+    --hds-spacing-md: 16px;
+    --hds-spacing-lg: 24px;
+    --hds-spacing-xl: 36px;
+    --hds-spacing-xxl: 48px;
+}
+
+// TODO: Determine best default spacing size
+$default-spacing: var(--hds-spacing-md); // set as Sass variable to not expose
+
+.hds-gutters {
+    // vertical (make default ?): (Find out most common use cases)
+    // &:not([class*="hds-gutters-horizontal"]), // yes/ no?
+    &-vertical {
```

Comment:
re. our BEM approach, we are not using this technique for concatenation, we just rely on flat declarations

Reply:
I can change it but I like to make use of sass nesting to keep things organized and avoid repeating the same selectors for shorter groups of styles.

(Btw, I didn't really follow BEM here since the "parent" class name doesn't really do anything. The other classes can be used on their own as helper classes.)

Reply:
Updated

Reply:
> I can change it but I like to make use of sass nesting

Absolutely! It's one of the most powerful thing of using a CSS pre-processor! What I meant using the `&` to construct BEM names/selectors.